.travis.yml

```diff
@@ -109,10 +117,16 @@ env:
     - MINCONDA_OSX="MacOSX-x86_64"
 
 install:
+  - export HOMEBREW_NO_AUTO_UPDATE=1
+  - if [[ "$BREW_AUTOUPDATE" && "$BREW_AUTOUPDATE" != 0 ]]; then
+      unset HOMEBREW_NO_AUTO_UPDATE;
+    fi
+
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       MINCONDA_OS=$MINCONDA_LINUX;
     elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-      HOMEBREW_NO_AUTO_UPDATE=1 brew install boost hdf5 eigen ninja;
+      brew install boost hdf5 eigen ninja;
+      if [ -z "$HOMEBREW_NO_AUTO_UPDATE" ]; then brew upgrade hdf5; fi;
```

Comment:
This seems brittle at best… does this rely on Travis caching an older `hdf5` version with the older XCode?

Reply:
I'm curious about that too? Could you explain @ferdonline

Reply:
I was actually proud this is slightly more stable... now, by default I set `HOMEBREW_NO_AUTO_UPDATE=1` in the top, so all installs provide the same version
However, atm to use hdf5 1.12 we need to force the upgrade. I wish there was a better solution, but nope. In either casethis only applies to the second MacOS test, wont affect any other setup